### PR TITLE
Tools changes to support bundles test

### PR DESCRIPTION
**Pull request by SergeyGaluzo**

### CODE REVIEW

File: tools/BlobRewriter/BlobRewriter.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <AssemblyName>Microsoft.Health.Internal.Fhir.BlobRewriter</AssemblyName>
+    <AssemblyName>Fhir.BlobRewriter</AssemblyName>
```
Comment (fhibf) [inline tools/BlobRewriter/BlobRewriter.csproj:5]: May we keep the original assembly name?

File: tools/EventsReader/EventsReader.csproj [hunk 2]
```diff
@@ -2,7 +2,7 @@
   
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <AssemblyName>Microsoft.Health.Internal.Fhir.EventsReader</AssemblyName>
+    <AssemblyName>Fhir.EventsReader</AssemblyName>
```
Comment (fhibf) [inline tools/EventsReader/EventsReader.csproj]: May we keep the original assembly name?

File: tools/Exporter/Exporter.csproj [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <AssemblyName>Microsoft.Health.Internal.Fhir.Exporter</AssemblyName>
+    <AssemblyName>Fhir.Exporter</AssemblyName>
```
Comment (fhibf) [inline tools/Exporter/Exporter.csproj]: The same here.

File: tools/Importer/BearerTokenHandler.cs [hunk 4]
```diff
@@ -150,5 +178,23 @@ public void Dispose(bool disposing)
 
             _disposed = true;
         }
+
+        private static void PrintClaimsFromJwt(string jwtToken)
```
Comment (fhibf) [inline tools/Importer/BearerTokenHandler.cs:182]: Do we need to print this information aways? Can we have a flag to enable this print or not. Just thinking that we can expose someone's information if that person decides to use this tool.

Comment (SergeyGaluzo) [reply tools/Importer/BearerTokenHandler.cs:182]: It is very handy for test tool. Yes.

File: tools/BlobRewriter/BlobRewriter.csproj (hunk 1 above)
Comment (SergeyGaluzo) [reply tools/BlobRewriter/BlobRewriter.csproj:5]: It is very inconvenient to have long exe name with all not distinct letters in front. This way I don't need to type much.

File: tools/Exporter/Exporter.csproj (hunk 3 above)
Comment (SergeyGaluzo) [reply tools/Exporter/Exporter.csproj]: Keeping original because name is referenced.

**Pull request merged**